build-llvm34.md

```diff
@@ -26,7 +26,8 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    **NOTE:** KLEE is currently tested on **Linux x86-64**, and might break on x86-32.
 
-   If you are using a recent Ubuntu (≥ 12.04 and ≤ 15.10, e.g. 14.04 LTS) or Debian, we recommend you to use the LLVM packages provided by LLVM itself. Use [LLVM Package Repository](http://llvm.org/apt/) to add the appropriate line to your `/etc/apt/sources.list`. As an example, for Ubuntu 14.04, the following lines should be added:  
+   If you are using a recent Ubuntu (≥ 12.04 and ≤ 15.10, e.g. 14.04
+   LTS), Debian or MacOS/Darwin, we recommend you to use the LLVM packages provided by LLVM itself. Use [LLVM Package Repository](http://llvm.org/apt/) to add the appropriate line to your `/etc/apt/sources.list`. As an example, for Ubuntu 14.04, the following lines should be added:  
```

Comment:
Good catch, there was no mention of MacOS, but that's the wrong place to add it.